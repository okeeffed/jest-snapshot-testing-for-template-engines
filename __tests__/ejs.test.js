const { helloTemplate } = require("../ejs");

describe("hello template file", () => {
  test("expect helloTemplate fn output to match snapshot", async () => {
    const data = { name: "World" };
    const str = await helloTemplate(data);

    expect(str).toMatchSnapshot();
  });
});
