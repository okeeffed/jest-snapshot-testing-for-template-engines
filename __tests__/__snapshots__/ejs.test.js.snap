// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`hello template file expect helloTemplate fn output to match snapshot 1`] = `
"<h2>Hello, World!</h2>
<p>This is something new that we want</p>
"
`;
